jobs/pr_gate/docker/monorail/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@PengTian0 
All the configuration files are read in during run_tests.py via mkcfg() into a 'running configuration' and as such each file has a top layer wrapper on it to keep them separated.  The tests then can access any parameter via fit_common fitcfg() or some of the shortcuts in there.  This particular file can be accessed via fit_common.fitrackhd().  So, we need to keep the wrapped rackhd-config header on it.
So, for now, since that isn't workable here, we can just copy the contents from config.json into this file via another PR as the default values (I can do that after the change to docker) and then work towards eliminating rackhd_defaults.json from the framework, and instead pull it from the deployed rackhd.
And like you say above, we aren't using much of it at all.  In our other amqp updates, I'm working to remove that reference you mention above. :-)